src/components/Signup/ExistingPodForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+// Custom Hook Imports
+import { useSession } from '@hooks';
+
+// MUI imports
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+/* Styles for MUI components */
+const cardStyle = {
+  padding: '8px',
+  margin: '8px',
+  boxShadow: '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)'
+};
+
+const formRowStyle = {
+  margin: '20px 0'
+};
+
+const textFieldStyle = {
+  margin: '8px'
+};
+
+const ExistingPodForm = () => {
+  const { login } = useSession();
+
+  const [oidcIssuer, setOidcIssuer] = useState('https://inrupt.net/');
+
+  const loginHandler = async (event) => {
+    event.preventDefault();
+
+    let redirectUrl = new URL(window.location.href);
+    redirectUrl = redirectUrl.toString();
+
+    localStorage.setItem('oidcIssuer', oidcIssuer);
+    await login({
+      oidcIssuer,
+      redirectUrl
+    });
+  };
+
+  return (
+    <Card style={cardStyle}>
```

Comment:
Not a big issue since it still works as intended, but MUI components have a dedicated `sx` attribute for custom CSS instead of `style` used for regular HTML elements. It lets MUI recognize any other potential themes that might be in place. https://mui.com/system/getting-started/the-sx-prop/